hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/snapshot/OMSnapshotMoveDeletedKeysResponse.java

```diff
@@ -156,7 +160,8 @@ private void processReclaimKeys(BatchOperation batchOp,
       throws IOException {
     for (SnapshotMoveKeyInfos dBKey : reclaimKeysList) {
       RepeatedOmKeyInfo omKeyInfos =
-          createRepeatedOmKeyInfo(dBKey.getKeyInfosList());
+          createRepeatedOmKeyInfo(dBKey.
+              getKeyInfosList(), bucketId);
```

Comment:
nit: unexpected line break

```suggestion
          createRepeatedOmKeyInfo(dBKey.getKeyInfosList(), bucketId);
```